volto_addon/{{ cookiecutter.addon_name }}/README.md

```diff
@@ -45,19 +50,45 @@ Start volto with:
 yarn start
 ```
 
-### Test it
+### Volto 18 and later
+
+Add `{{ cookiecutter.npm_package_name }}` to your `package.json`:
+
+```json
+"dependencies": {
+    "{{ cookiecutter.npm_package_name }}": "*"
+}
+```
+
+Add `{{ cookiecutter.npm_package_name }}` to your `volto.config.js`:
+
+```javascript
+const addons = ['{{ cookiecutter.npm_package_name }}'];
+```
```

Comment:
Isn't there a missing step to install the new package after adding it to package.json?

Reply:
`make install` probably.